### PR TITLE
creating classes

### PR DESCRIPTION
**Pull request by Amybarba**
Creating Classes and prototype refactor

### CODE REVIEW

File: assignments/lambda-classes.js [hunk 1]
```diff
@@ -1 +1,73 @@
...
+    constructor(props) {
+        super(props);
+this.gradClassName = props.gradClassName;
+this.favInstructor = props.favInstructor;
+}
+
+standup(channel) {
+    console.log(`${this.name} announces to ${channel}, @channel standy times!`);
+}
+}
```
Comment (AsaOfDiamonds) [inline assignments/lambda-classes.js:66]: you have an extra curly brace so the next part of your code is outside of the ProjectManager

File: assignments/lambda-classes.js [hunk 2]
```diff
@@ -1 +1,73 @@
...
+
+debugsCode(student, subject) {
+    console.log(`${this.name} debugs ${studen.name}'s code on ${subject}`);
```
Comment (AsaOfDiamonds) [inline assignments/lambda-classes.js:69]: you have a typo here studen.name should be student.name

File: assignments/lambda-classes.js [hunk 3]
```diff
@@ -1 +1,73 @@
...
+}
+}
+
+consgt fred =
```
Comment (AsaOfDiamonds) [inline assignments/lambda-classes.js:73]: I will let you figure out what happened here lol

Comment (AsaOfDiamonds) [reply assignments/lambda-classes.js:73]: const fred = new Instructor({
  name: 'Fred',
  location: 'Bedrock',
  age: 37,
  gender: 'male',
  favLanguage: 'JavaScript',
  specialty: 'Front-end',
  catchPhrase: `Don't forget the homies`
});

const bill = new ProjectManager({
  name: 'Bill',
  location: 'Michigan',
  age: 32,
  gender: 'female',
  gradClassName: 'CS1',
  favInstructor: 'Luis',
  favLanguage: 'JavaScript',
  specialty: 'Auth',
  catchPhrase: 'Redux Rocks!'
});

const figg = new Student({
  name: 'Figgy',
  location: 'Charleston',
  age: 37,
  gender: 'male',
  previousBackground: 'Fork Lift Operator',
  className: 'CS11',
  favSubjects: ['Html', 'CSS', 'JavaScript']
});

fred.speak();
fred.demo('variables');
fred.grade(figg, 'redux');

figg.speak();
figg.listsSubjects();
figg.PRAssignment('Redux');
figg.sprintChallenge('Pre-proccessing');

bill.speak();
bill.demo('Redux');
bill.grade(figg, 'const vs let');
bill.debugsCode(figg, 'DS-Algos');
bill.standUp('#Code-Allstars');

**Review by AsaOfDiamonds — APPROVED** [review]
👍  I left some comments in your code make sure to read them.

**Comment by Amybarba** [comment]
In looking closer at my VScode I must have gone back after I submitted and fixed most of these, but did not commit after that

**Pull request merged**